hawkbit-device-simulator/src/main/resources/application.properties

```diff
@@ -39,4 +39,9 @@ spring.rabbitmq.host=localhost
 spring.rabbitmq.port=5672
 spring.rabbitmq.dynamic=true
 
+## Configuration for basic auth
+hawkbit.device.simulator.auth.enabled=false
+hawkbit.device.simulator.auth.user=""
```

Comment:
IMO, add some values for USER and PWD although it's of no use by default

Reply:
Done